test/unit/skin-test.js

```diff
@@ -0,0 +1,11 @@
+import _ from 'test/underscore';
+import { normalizeSkin } from 'utils/skin';
+
+describe('Skin Customization', function() {
+
+    it('should normalize an empty skin config', function() {
+        const skinColors = normalizeSkin({});
+        expect(typeof skinColors).to.equal('object');
+        expect(_.size(skinColors)).to.equal(4);
+    });
+});
```

Comment:
Can you add a test to make sure new styles are not added to the DOM when custom setting are not passed to `normalizeSkin` and `handleColorOverrides`?